support-frontend/assets/helpers/checkouts.js

```diff
@@ -206,7 +206,9 @@ function getAvailablePaymentRequestButtonPaymentMethod(
   const switchKey = switchKeyForContributionType(contributionType);
   if (result && result.applePay === true && isSwitchOn(`${switchKey}.stripeApplePay`)) {
     return 'StripeApplePay';
-  } else if (result && result.applePay === false && isSwitchOn(`${switchKey}.stripePaymentRequestButton`)) {
+  } else if (result && result.googlePay === true) {
+    return 'StripeGooglePay';
+  } else if (result) {
```

Comment:
I guess here we'll actually want something like: 

```ts
} else if (result && result.googlePay === true && isSwitchOn(`${switchKey}.stripeGooglePay`)) {
```

And to add the switch check back for `StripePaymentRequestButton`. @tomrf1 how do I go about adding a new switch?

Reply:
Discussed IRL: it's a bit of a can of worms adding `StripeGooglePay`, so we're going to ignore this for now and use a work around